src/OpenSage.Game/Data/W3d/W3dEmitterLineProperties.cs

```diff
@@ -21,7 +21,7 @@ internal static W3dEmitterLineProperties Parse(BinaryReader reader, W3dParseCont
             {
                 var result = new W3dEmitterLineProperties
                 {
-                    Flags = reader.ReadUInt32AsEnum<W3dEmitterLineFlags>(),
+                    Flags = (W3dEmitterLineFlags)reader.ReadUInt32(),
```

Comment:
Why this change? Which new values have you found?

Reply:
Looks like I changed this because it was failing to parse based on a new value. I wasn't sure what the flag was for. I'll revert it and add as a UnknownENB value in the enum. Its value is: ... (0x00000002 << 24) + 1. It is possible this was corruption. I have only found this in a few enb assets.